### PR TITLE
New Features updates for PowerMax 

### PR DESCRIPTION
**Pull request by shanmydell**
# Description
New Features updates for PowerMax 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/646
https://github.com/dell/csm/issues/609 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rsedlock1958 — DISMISSED** [review]
I have reviewed this. No grammatical edits are required.

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by tssushma — APPROVED** [review]
lgtm

File: content/docs/_index.md [hunk 1]
```diff
@@ -64,19 +64,19 @@ CSM is made up of multiple components including modules (enterprise capabilities
 
 ## CSM Supported Modules and Dell CSI Drivers
 
-| Modules/Drivers                                                                              | CSM 1.5 | [CSM 1.4](../v1/) | [CSM 1.3.1](../v2/) | [CSM 1.2.1](../v3/) |
+| Modules/Drivers                                                                              | CSM 1.6 | [CSM 1.5.1](../v1/) | [CSM 1.4](../v2/) | [CSM 1.3](../v3/) |
 | -------------------------------------------------------------------------------------------- | :-----: | :---------------: | :---------------: | :-----------------: |
 | [Authorization](https://hub.docker.com/r/dellemc/csm-authorization-sidecar)                  | v1.5.0  |      v1.4.0       |      v1.3.0       |       v1.2.0        |
```
Comment (santhoshatdell) [inline content/docs/_index.md:69]: Hi @shanmydell 
Please correct the versions of CSM modules for previous releases. They need to be updated accordingly when a new release/column is added.

**Pull request merged**